services/wallet/erc20/erc20.go

```diff
@@ -0,0 +1,353 @@
+// Code generated - DO NOT EDIT.
```

Comment:
How is it generated? Do we have gogenerate for it?

Reply:
i described in README.md which solidity snippet was used to generate this file. i generated it with abigen. i can try to make gogenerate statement for it

Reply:
added file with gogenerate statement